### PR TITLE
Fix link to linter.md

### PR DESCRIPTION
**Pull request by astroash**
Fix link to linter.md in week 5 REAME.md

### CODE REVIEW

File: coursebook/week-5/README.md [hunk 1]
```diff
@@ -4,72 +4,72 @@
 
 #### Day 1
 
-- 10:00 - 10:15 - Intro to Node week 2
+* 10:00 - 10:15 - Intro to Node week 2
```
Comment (ZooeyMiller) [inline coursebook/week-5/README.md]: why the change from `-` to `*` for ULs? 

Is this prettier's doing?

Comment (astroash) [reply coursebook/week-5/README.md]: Yeah, sorry about that 😞

**Comment by astroash** [comment]
Eep, actually latest eslint works differently to mine! Gunna have to rewrite this!
